e2e/scenario_test.go

```diff
@@ -0,0 +1,983 @@
+package e2e
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/agentbaker/pkg/agent/datamodel"
+	"github.com/Azure/agentbakere2e/config"
+	"github.com/Azure/agentbakere2e/toolkit"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerservice/armcontainerservice"
+)
+
+func Test_azurelinuxv2(t *testing.T) {
```

Comment:
would it be possible to split this file into a few more, by publisher name maybe ?  

1- helps with keeping the same type of OS together, 
2- helps with reducing the file size.

Reply:
🤷 I don't have a fear of big files. I found it convenient to have 20+ similar structures in a single file.

Do you have a strong opinion on it? Or maybe someone else has preference?